src/test/java/teofilursan/seleniumpart2/tests/seleniumpart1/SeleniumExampleTest.java

```diff
@@ -1,4 +1,4 @@
-package teofilursan.tests.seleniumpart1;
+package teofilursan.seleniumpart2.tests.seleniumpart1;
```

Comment:
Please leave the same package as it was before package teofilursan.tests.seleniumpart1; don't move it to package teofilursan.seleniumpart2.tests.seleniumpart1; Thanks!

Reply:
ok, numai ca nu stiu cum sa fac asta.

Reply:
Drag and drop package tests to teofilursan -> 
![image](https://user-images.githubusercontent.com/18215854/215508839-d5d4cc63-ea02-4cd9-9876-7b5a7f89aca4.png)